docker-entrypoint.sh

```diff
@@ -10,7 +10,8 @@ fi
 # allow the container to be started with `--user`
 if [ "$1" = 'redis-server' -a "$(id -u)" = '0' ]; then
 	find . \! -user redis -exec chown redis '{}' +
-	exec gosu redis "$0" "$@"
+	# setpriv is builtin in busybox so invoke as /usr/bin/setpriv
+	exec /usr/bin/setpriv --reuid=redis -- "$0" "$@"
```

Comment:
```suggestion
	exec setpriv --reuid=redis --regid=redis --clear-groups -- "$0" "$@"
```

```console
$ sudo setpriv --reuid=nobody id
uid=65534(nobody) gid=0(root) groups=0(root)
$ sudo setpriv --reuid=nobody --regid=nogroup id
setpriv: --[re]gid requires --keep-groups, --clear-groups, --init-groups, or --groups
$ sudo setpriv --reuid=nobody --regid=nogroup --clear-groups id
uid=65534(nobody) gid=65534(nogroup) groups=65534(nogroup)
```

(also we do not want to specify an explicit path -- that's what `$PATH` is for)

Reply:
/bin is before /usr/bin in PATH. /bin/setpriv is busybox. What do you recommend?